exercise_08/formatStringByWordsLength.js

```diff
@@ -5,14 +5,13 @@
  * input: string (sentence) 
  * output: string
  */
+function formatStringByWordsLength(sentence) {
+    const words = sentence.trim().split(/\s+/);
+    if (words.length > 5) {
```

Comment:
Avoid magic numbers. 

```suggestion
    if (words.length > THRESHOLD) {}
```

- [ ] Extract number to constant